core/designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/component/DropdownMenu.kt

```diff
@@ -80,6 +110,39 @@ fun <T> NiaDropdownMenuButton(
     }
 }
 
+/**
+ * Internal Now in Android dropdown menu button content layout for arranging the text label and
+ * trailing icon.
+ *
+ * @param text The button text label content.
+ * @param trailingIcon The button trailing icon content. Pass `null` here for no trailing icon.
```

Comment:
Default to `null` instead to avoid having to explicitly pass `null`?

Reply:
Done